remote-api/src/test/java/org/alfresco/rest/api/tests/AbstractBaseApiTest.java

```diff
@@ -224,6 +224,11 @@ protected String getRequestArchivedContentDirectUrl(String nodeId)
         return URL_DELETED_NODES + "/" + nodeId + "/" + REQUEST_DIRECT_ACCESS_URL;
     }
 
+    protected String getRequestArchivedRenditonContentDirectUrl(String nodeId, String renditionID)
```

Comment:
This needs a rendition id in the endpoint request:
deleted-nodes/{{nodeId}}/renditions/{{renditionId}}/request-direct-access-url

Reply:
sorry, didn't spot that had already changed